### DISCUSSION TITLE
shall we update the matplotlib version used in this project?

### DISCUSSION THREAD

**Discussion by chansigit** [aristoteleo/dynamo-release#683] [General]
The required matplotlib is rather old and incompatible with other anndata ecosystem packages.

**Comment by chansigit** [comment]
New version of matplotlib also provides handy tools for vector field visualization.
https://matplotlib.org/stable/plot_types/arrays/index.html

**Reply by Sichao25** [reply]
Great suggestion! We will update this soon.

**Comment by chansigit** [comment]
done.
